nuxt.config.ts

```diff
@@ -129,6 +129,7 @@ exports.randomFillSync = randomFillSync`
           app_id: ''
         }
       },
+      IS_MAINTENANCE: process.env.IS_MAINTENANCE !== undefined,
```

Comment:
If we set `IS_MAINTENANCE=` in `.env`, would it be `''` or `undefined`?

Reply:
it is '', updated